api/src/main/java/org/openmrs/api/db/hibernate/HibernatePatientDAO.java

```diff
@@ -211,6 +211,23 @@ public List<Patient> getPatients(String query, boolean includeVoided, Integer st
 		return new ArrayList<>(patients);
 	}
 	
+	/**
+	 * @see org.openmrs.api.PatientService#getPatientByIdentifier(String, String, List, boolean, boolean, boolean)
+	 * <strong>Should</strong> return only exact match 
+	 */
+	@Override
+	public List<Patient> getPatientsByIdentifier(String name, String identifier, List<PatientIdentifierType> identifierTypes,
+		boolean matchIdentifierExactly) throws DAOException {
+		Criteria criteria = sessionFactory.getCurrentSession().createCriteria(Patient.class);
+		criteria = new PatientSearchCriteria(sessionFactory, criteria).prepareCriteria(name, identifier, identifierTypes,
+			matchIdentifierExactly, false, false);
```

Comment:
Is it possible to put it on the same line.

Reply:
It's ok with me but just one thing; the file is indented as proposed in this PR. If we are to change the indention here this may  call for indenting other methods as well. Let me know what you think?